src/libstd/sys/unix/fs.rs

```diff
@@ -20,6 +20,10 @@ use libc::{stat64, fstat64, lstat64, off64_t, ftruncate64, lseek64, dirent64, re
 use libc::fstatat64;
 #[cfg(any(target_os = "linux", target_os = "emscripten", target_os = "android"))]
 use libc::dirfd;
+// We only use struct `statx`, not the function `statx`.
+// Instead, use `syscall` to check if it is available at runtime.
+#[cfg(target_os = "linux")]
+use libc::{statx, makedev};
```

Comment:
Since this is only used in the `try_statx` function, could this `use` be moved down there?

Reply:
It will collide `fn statx` in the function body.
I think we can simply remove the `use` and refer them with `libc::`.